util/occamygen.py

```diff
@@ -455,33 +566,22 @@ def main():
         context="soc",
         node=am_soc_wide_xbar)
 
-    for i in range(nr_s1_quadrants):
-        soc_wide_xbar.add_output_symbolic("s1_quadrant_{}".format(i),
-                                          "s1_quadrant_base_addr",
-                                          "S1QuadrantAddressSpace")
-        soc_wide_xbar.add_input("s1_quadrant_{}".format(i))
-
-    for i in range(8):
-        soc_wide_xbar.add_output_entry("hbm_{}".format(i), am_hbm[i])
-
-    for i in range(nr_s1_quadrants+1):
-        soc_wide_xbar.add_input("hbi_{}".format(i))
-    soc_wide_xbar.add_output_entry("hbi_{}".format(nr_s1_quadrants), am_hbi)
-
-    soc_wide_xbar.add_input("soc_narrow")
+    # Default port: HBI (always escalate "upwards" in hierarchy -> off-chip)
+    soc_wide_xbar.add_output_entry("hbi", am_hbi)
+    soc_wide_xbar.add_output_entry("hbm_xbar", am_hbm_xbar)
+    soc_wide_xbar.add_output_entry("quadrant_inter_xbar", am_quadrant_inter_xbar)
     soc_wide_xbar.add_output_entry("soc_narrow", am_soc_narrow_xbar)
-
-    # TODO(zarubaf): PCIe should probably go into the small crossbar.
-    soc_wide_xbar.add_input("pcie")
-    soc_wide_xbar.add_output_entry("pcie", am_pcie)
```

Comment:
In the FPGA designs we (mis-)use the PCI ports for peripherals (QSPI, HBM config, ...). Could we not remove this and make it parametrizable in `occamy_cfg.hjson`?

Reply:
The PCI port is still present and now connected to the narrow X-bar. Do you need it on the wide x-bar? If so, you can instead use the single HBI port, which now goes directly to the wide X-bar; there is no difference between this new HBI connection and the old PCI connection.

Reply:
didn't catch that, narrow is fine thanks!